pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -170,20 +170,24 @@ func initHandler() {
 	}
 }
 
-func setCachedInterface(ifconf *api.Interface) error {
+func getInterfaceCacheFile(name string) string {
+	return fmt.Sprintf(interfaceCacheFile, name)
+}
+
+func setCachedInterface(name string, ifconf *api.Interface) error {
 	buf, err := json.MarshalIndent(&ifconf, "", "  ")
 	if err != nil {
 		return fmt.Errorf("error marshaling interface cache: %v", err)
 	}
-	err = ioutil.WriteFile(interfaceCacheFile, buf, 0644)
+	err = ioutil.WriteFile(getInterfaceCacheFile(name), buf, 0644)
```

Comment:
Nothing which you introduced in this PR, but why are we writing the config to a file? If virt-launcher crashes the pod will go to finished. We can as well keep it in memory.

Reply:
That I don't know. As you can see, I am still pretty bad at broad picture of all components and how they interact. :) I will look into it.